tests/presentation-definition/multi-credential-array.json

```
@@ -0,0 +1,88 @@
+{
+  "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
+  "name": "Identity and Qualifications Verification",
+  "purpose": "We need to verify your identity and qualifications",
+  "format": {
+    "jwt_vp": {
+      "alg": ["EdDSA", "ES256K"]
+    },
+    "jwt_vc": {
+      "alg": ["EdDSA", "ES256K"]
+    },
+    "ldp_vc": {
+      "proof_type": ["Ed25519Signature2018", "EcdsaSecp256k1Signature2019"]
+    },
+    "ldp_vp": {
+      "proof_type": ["Ed25519Signature2018", "EcdsaSecp256k1Signature2019"]
+    }
+  },
+  "input_descriptors": [
+    {
+      "id": "combined_credential",
+      "name": "Identity and Education Credential",
+      "purpose": "Please provide a credential that includes both identity and educational information",
+      "constraints": {
+        "fields": [
+          {
+            "path": ["$.type", "$.vc.type"],
+            "filter": {
+              "type": "array",
+              "contains": {
+                "const": "IdentityCredential"
+              }
+            }
+          },
+          {
+            "path": ["$.type", "$.vc.type"],
+            "filter": {
+              "type": "array",
+              "contains": {
+                "const": "EducationalCredential"
+              }
+            }
+          },
+          {
+            "path": ["$.credentialSubject.firstName", "$.vc.credentialSubject.firstName"],
+            "purpose": "The credential must contain the holder's first name",
+            "filter": {
+              "type": "string",
+              "minLength": 1
```

NABD but I don't think the `minLength: 1` filter is necessary
```suggestion
              "type": "string"
```